builders/deploy/dapp.md

```diff
@@ -142,6 +142,48 @@ Finally, [generate and edit](/builders/build/customize/customizing-chain-specs/#
 - `para_id` - within this custom flow, a pre-registered parachain id is required
 - `is_ethereum` - to `true` if exposing Ethereum compatible RPC endpoints is needed
 
+And, depending on whether you are deploying a Snap Appchain or a dedicated one, also adjust these attributes:
+
+=== "Snap Appchains"
```

Comment:
this should just be "Snap Appchain" to align with the other tabbed element in the check balances section, or that one should be changed to "Snap Appchains" no preference really they should just be the same 🙂